setup.py

```diff
@@ -6,8 +6,10 @@
 
 import os.path
 import sys
-
+import glob
```

Comment:
run `isort` tool please

Reply:
isort will not only move up `import glob` but also swap the two lines below:
```
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "habitat"))
from version import VERSION  # noqa
```
Which causes installation failure and causes failure in CI test. 
Ignoring the swap right now.